demo/c/README.md

```diff
@@ -1,42 +1,82 @@
 # Compatibility
 
-You need a C99-compatible compiler to build these demos. The microphone based demo can only run on Linux-based systems
-such as `Ubuntu`, `Raspberry Pi`, and `BeagleBone` as it depends on `ALSA`.
+You need a C99-compatible compiler to build these demos. The microphone based demo can run on
+any platform that supports `miniaudio`. 
 
-# Usage
+#Usage
```

Comment:
do you need this?
if yes, it should be `# Usage` and then there are two "usage"s in this doc